src/use_case/create_event/CreateEventInputData.java

```diff
@@ -1,4 +1,74 @@
 package use_case.create_event;
 
 public class CreateEventInputData {
+    private final String ownerUsername;
+    private final String eventName;
+    private final String location;
+    private final String time;
+    private final String type;
+    private final String description;
+    //For privacy and inviteOnly, I was thinking of using a checkbox in the create event view to tick these options off.
+    private final Boolean privacy;
+    private final Boolean inviteOnly;
+    private final Integer capacity; //Text box for capacity will only allow numbers to be inputted.
+    private final Integer ageRestriction;
+    private final String sexRestriction;
+
+    private CreateEventInputData(String owner, String eventName, String location, String time, String type, String description,
```

Comment:
Good use of method overloading to separate different cases under the same use case!